apps/api/package.json

```diff
@@ -23,17 +23,20 @@
     "@nestjs/common": "8.0.0",
     "@nestjs/core": "8.0.0",
     "@nestjs/platform-express": "8.0.0",
-    "reflect-metadata": "0.1.13",
-    "rimraf": "3.0.2",
-    "rxjs": "7.2.0"
+    "@vendia/serverless-express": "4.5.2",
+    "aws-lambda": "^1.0.7",
+    "reflect-metadata": "^0.1.13",
+    "rimraf": "^3.0.2",
+    "rxjs": "^7.2.0"
   },
   "devDependencies": {
     "@nestjs/cli": "^8.0.0",
     "@nestjs/schematics": "^8.0.0",
     "@nestjs/testing": "^8.0.0",
+    "@types/aws-lambda": "^8.10.89",
     "@types/express": "^4.17.13",
     "@types/jest": "27.0.2",
-    "@types/node": "^16.0.0",
+    "@types/node": "^17.0.8",
```

Comment:
v17? We need to coordinate on node version to use.

Reply:
not sure how it got updated. ill remove this